app/src/test/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerStateTest.kt

```diff
@@ -148,6 +148,28 @@ class MessageComposerInnerStateTest {
         assertEquals(null, state.mentionQueryFlowState.value)
     }
 
+    // case was found by manual testing
+    @Test
+    fun `when mention symbol is at the begin of new line, then mention is requested`() = runTest {
+        val state = createState(context)
+        state.setMessageTextValue(textFieldValueWithSelection("some text\n"))
+
+        state.setMessageTextValue(textFieldValueWithSelection("some text\n@"))
+
+        assertEquals("", state.mentionQueryFlowState.value)
+    }
+
+    // case was found by manual testing
+    @Test
+    fun `when cursor is at the begin of new line, and add mention btn clicked, then mention is requested`() = runTest {
```

Comment:
I'ld suggest to use `given, when, then` instead
`given cursor is at the begin of new line, when add mention button clicked, then mention is requested`